storybook/stories/ui-charts/manchetteWithSpaceTimeChart/rectangle-zoom.stories.tsx

```diff
@@ -69,11 +70,7 @@ const ManchetteWithSpaceTimeWrapper = ({
         onScroll={handleScroll}
       >
         <Manchette {...manchetteProps} />
-        <div
-          className="space-time-chart-container w-full sticky"
-          ref={spaceTimeChartRef}
-          style={{ bottom: 0, left: 0, top: 2, height: `${DEFAULT_HEIGHT - 6}px` }}
```

Comment:
really nice not to have to give all these specific dimensions and have all this mess simplified !